tests/sentry/integrations/slack/webhooks/commands/test_help.py

```diff
@@ -19,16 +22,201 @@ def assert_unknown_command_text(data: SlackBody, unknown_command: Optional[str]
+                                "text": "Here are the commands you can use. Commands not working? Re-install the app!",
+                            },
+                        },
+                        {
+                            "type": "section",
+                            "text": {"type": "mrkdwn", "text": "*Direct Message Commands:*"},
+                        },
+                        {
+                            "type": "section",
+                            "text": {
+                                "type": "mrkdwn",
+                                "text": "`/sentry help`: View this list of commands.\n`/sentry link`: Link your Slack identity to your Sentry account to receive notifications. You'll also be able to perform actions in Sentry through Slack.\n`/sentry unlink`: Unlink your Slack identity from your Sentry account.",
+                            },
+                        },
+                        {
+                            "type": "section",
+                            "text": {"type": "mrkdwn", "text": "*Channel Commands:*"},
+                        },
+                        {
+                            "type": "section",
+                            "text": {
+                                "type": "mrkdwn",
+                                "text": "`/sentry link team`: Get your Sentry team's issue alert notifications in the channel this command is typed in.\n`/sentry unlink team`: Unlink a team from the channel this command is typed in.",
+                            },
+                        },
+                        {"type": "section", "text": {"type": "mrkdwn", "text": "*Contact:*"}},
+                        {
+                            "type": "section",
+                            "text": {
+                                "type": "mrkdwn",
+                                "text": "Let us know if you have feedback: ecosystem-feedback@sentry.io",
+                            },
+                        },
+                        {"type": "divider"},
+                        {
+                            "type": "section",
+                            "text": {
+                                "type": "mrkdwn",
+                                "text": "Just want to learn more about Sentry? Check out our <https://docs.sentry.io/product/integrations/notification-incidents/slack/|documentation>.",
+                            },
+                        },
+                    ]
+                },
+            )
         data = self.send_slack_message("invalid command")
         assert_unknown_command_text(data, "invalid command")
 
+    @responses.activate
     def test_help_command(self):
+        if SiloMode.get_current_mode() == SiloMode.CONTROL:
+            responses.add(
+                method=responses.POST,
+                url="http://testserver/extensions/slack/commands/",
+                json={
+                    "blocks": [
```

Comment:
Should we consolidate this payload into a single definition? Either as a python dict or a json fixture file that could be loaded?